### DISCUSSION TITLE
Join two dataframes with different number of rows

### DISCUSSION THREAD

**Discussion by ghost** [vaexio/vaex#1905] [Q&A]
Hello again,
In my case i have two dataframes. The first one has records from a simulation solving PDEs in a sparse grid and the second one has records for a dense grid. The two grids have different step but some grid points are the same.

I am trying to see what is the difference between the two of them in certain geo area (the same grid point-LAT,LON) and the same grid points. 

I want to do that by joining them based on the same columns (LAT,LOT) and just figuring RMSE or simple substraction. But i find that the two datasets after filtering (like Jovan showed in a previous post) have different number of rows (maybe dublicates exist : / )

Any ideas how i can deal with that??

Dense grid
![image](https://user-images.githubusercontent.com/39860343/153409314-1cf47106-1da8-462c-94e0-b029c5740dec.png)
![image](https://user-images.githubusercontent.com/39860343/153410828-bb0753ce-ee3e-41a5-9bc1-3b97e981d695.png)

Sparse grid
![image](https://user-images.githubusercontent.com/39860343/153410033-bc33ca45-5435-404f-ad9d-7b33e6321475.png)
![image](https://user-images.githubusercontent.com/39860343/153410770-b863807e-e195-47a1-aaa8-557c9f07eeaf.png)

**Comment by maartenbreddels** [comment]
Hi,

it's unclear to me what the question is. Maybe you can elaborate.

Regards,

Maarten

**Reply by JovanVeljanoski** [reply]
you can join allowed for duplicates (it is a an argument of join). 
It is better to always include a runnable example with your question, otherwise hard for us to help.

**Reply by ghost** [reply]
Hey guys, i have already fixed it, thanx for answering. Thanx for everything.
Regards,
J.
